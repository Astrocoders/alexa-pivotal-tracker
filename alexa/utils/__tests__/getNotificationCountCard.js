const getNotificationsCountCard = require('../getNotificationsCountCard')

test('it should handle count when it is 0 properly', () => {
  const cardDef = getNotificationsCountCard(0)

  expect(cardDef).toMatchSnapshot()
})

test('it should handle count when it is 1 properly', () => {
  const cardDef = getNotificationsCountCard(1)

  expect(cardDef).toMatchSnapshot()
})

test('it should handle count when it is more than 1 properly', () => {
  const cardDef = getNotificationsCountCard(10)

  expect(cardDef).toMatchSnapshot()
})
