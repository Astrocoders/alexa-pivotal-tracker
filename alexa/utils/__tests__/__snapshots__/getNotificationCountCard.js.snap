// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should handle count when it is 0 properly 1`] = `
Object {
  "content": "You don't have any new notifications. Well done!",
  "title": "Pivotal Tracker",
  "type": "Simple",
}
`;

exports[`it should handle count when it is 1 properly 1`] = `
Object {
  "content": "You have 1 new notification",
  "title": "Pivotal Tracker",
  "type": "Simple",
}
`;

exports[`it should handle count when it is more than 1 properly 1`] = `
Object {
  "content": "You have 10 new notifications",
  "title": "Pivotal Tracker",
  "type": "Simple",
}
`;
